apps/dashboard/src/app/knowledge-box/knowledge-box-process/knowledge-box-processes.component.scss

```diff
@@ -41,6 +41,10 @@ h2 {
   }
 }
 
+.toggle-all {
+  border-bottom: 1px solid $color-neutral-lighter;
+}
```

Comment:
with the pa-separator you won't need this

Reply:
Fixed